packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -741,6 +791,57 @@ QUnit.module('loop', {
         delete this.animationStartAction;
     }
 }, () => {
+    QUnit.test('when only one item is visible, swipe action does not move the current visible item', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false }
+            ],
+            loop: true
+        });
+        const $itemContainer = $multiView.find(toSelector(MULTIVIEW_ITEM_CONTAINER_CLASS));
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(-0.1);
+        assert.strictEqual(position($itemContainer), 0, 'container did not move');
+        pointer.swipeEnd();
+    });
+
+    QUnit.test('when swiping left on the first item, show last visible item', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false },
+                { text: '3', visible: true }
+            ],
+            loop: true
+        });
+        const instance = $multiView.dxMultiView('instance');
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(-0.5).swipeEnd(-1);
+
+        assert.strictEqual(instance.option('selectedIndex'), 2, 'Correct item is shown after swiping left');
+    });
+
+    QUnit.test('when swiping right on the last item, show first visible item', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false },
```

Comment:
Let's make `visible: false` for the first item and `true` for the second one?
Because swiping from the last item to the first one is a default behavior, your test does not check how it works with hidden items

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)